common/flatpak-dir.h

```diff
@@ -80,9 +80,51 @@ typedef struct
 
 void         flatpak_related_free (FlatpakRelated *related);
 
+
+/* The collection state represent the state of the remote at a particular
+   time, including the summary file and the metadata (which may be from
+   the summary or from a branch. We create this once per highlevel operation
+   to avoid looking up the summary multiple times, but also to avoid races
+   if it happened to change in the middle of the operation */
+typedef struct
+{
+  char *remote;
```

Comment:
s/`remote`/`remote_name`/? I’d expect something called `remote` to be an `OstreeRemote` or `FlatpakRemote` object.